plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -31,10 +28,13 @@ import transformer, {
 } from '../transformers';
 import { docStorageURL } from '../../config';
 import URLFormatter from '../urlFormatter';
+import { TechDocsNotFound } from './TechDocsNotFound';
+import { TechDocsPageWrapper } from './TechDocsPageWrapper';
 
 const useFetch = (url: string) => {
```

Comment:
Now that this returns an Error (and might have before still have returned an error if e.g. `fetch` can't access the URL) we could be more explicit with the types:

```suggestion
const useFetch = (url: string): string | Error => {
```